gma/integration_test/src/integration_test.cc

```diff
@@ -228,11 +234,11 @@ void FirebaseGmaTest::SetUpTestSuite() {
 }
 
 void FirebaseGmaTest::TearDownTestSuite() {
-  // Workaround: GMA does some of its initialization in the main
-  // thread, so if you terminate it too quickly after initialization
-  // it can cause issues.  Add a small delay here in case most of the
-  // tests are skipped.
-  ProcessEvents(1000);
+  // GMA does some of its initialization in the main thread, so if you terminate
+  // it before initialization has completed, it can cause issues. So wait for
+  // any pending GMA initialization to be completed before calling terminate.
+  auto initialize_future = firebase::gma::InitializeLastResult();
+  WaitForCompletion(initialize_future, "gma::Initialize");
```

Comment:
This could just be WaitForCompletion(firebase::gma::InitializeLastResult(), "gma::Initialize") for brevity.

Reply:
Done